packages/backend/package.json

```diff
@@ -34,6 +34,7 @@
     "dockerode": "^3.2.0",
     "express": "^4.17.1",
     "knex": "^0.21.1",
+    "pg": "^8.3.0",
```

Comment:
8 here, 10 in the image - is that expected?

Reply:
This is `npm` package, and before is `docker` image, idk if think they are THAT correlated?

Reply:
Me neither, just checking

Reply:
Checked, they are not correlated, all good